terraform/stages/gke/providers.tf

```diff
@@ -28,7 +28,7 @@ terraform {
     }
     helm = {
       source  = "hashicorp/helm"
-      version = ">= 2.5.1"
+      version = ">= 2.7.0"
```

Comment:
We need to test this.

Reply:
The helm version is 2.7.0 in terraform/modules/ingress/main.tf
I have been testing it by creating a solutions template installation from scratch